### ISSUE TITLE
Does this viewer can display FsmTemplate ?

### ISSUE THREAD

**Issue by Soft-me** [nesrak1/FSMViewAvalonia#12]
I noticed there is an Action called SetFsmGameObject mentioned a fsm named "health_manager_enemy", but I couldn't find it.
And this fsm is a FSMTemplate, so is there any way to extract them ?

**Comment by nesrak1** [comment]
No, I don't read FSMTemplate right now. While the format looks basically the same as regular FSMs, I have a few speed hacks that make adding another type somewhat difficult to add at the moment. If I have free time, I can add this if it's important.

**Comment by Soft-me** [comment]
Thanks for replying. You mean you are too busy to add this function, right?

Anyway, I try to extract FSMTemplate by myself, but I don't know how to start. You say the format looks the same as regular FSMs, so I guess I can just find where these template are and problem solved. But I don't know where they are. Playmaker.dll or resources.assets? I don't know.

I try to use `AssetTypeValueField fsmTemplate = baseField.Get("fsmTemplate");` in function `LoadFSM` of class `FSMLoader`.  And debug shows it has two children, so I guess it's not the answer.

This tool is useful for me truly. And I think it will be more powerful if it can read FSMTemplate, considering many PlayMakerFSM use FSMTemplate.

If you have free time to add this now, that is great ! If not , could you please tell me where I should start. 

Anyway, thanks to create this useful tool~

**Comment by nesrak1** [comment]
I had a bit of free time so I just pushed the changes for FsmTemplate support. Let me know if it works. (You'll have to build it yourself since I don't have any builds for this repo)

**Comment by Soft-me** [comment]
It works! 
The only problem is it doesn't display all FSMTemplate in resource.assets .
I'm saying it because some template from a project I extract doesn't display in resource list.
Maybe some of them store in another place ...

**Comment by nesrak1** [comment]
I don't think which file it's in matters so it's probably in a different file than you expect. Hopefully sometime soon we can have FSM searching / MonoBehaviour searching somewhere.
